app/pages/options.html

```diff
@@ -29,15 +29,28 @@
       <div id="datapurgemodal" class="modal">
         <div class="modal-content">
           <p>
-            Opting out will delete the study data from your browser.
-            If you want to keep them you should
-            <a href="studydataview.html" target="_blank">make a copy</a>
+            Opting out will end your participation and delete the study data from your browser.
+            You can also choose to remove all your data from our study.
+            However we appreciate if you let us use previously sent data.
+            <br>
+            If you want to keep a copy you should
+            <a href="studydataview.html" target="_blank">make one</a>
             before continuing.
           </p>
-          <span id="purgeCont" class="button btn-red" role="button">Delete data and opt-out</span>
-          <span id="purgeAbort" class="button btn-gray" role="button">Opt-out later</span>
+          <p id="deleteServer-box">
+            <label for="deleteServer">Remove data from study</label>
+            <input type="checkbox" id="deleteServer" name="deleteServer">
+          </p>
+          <p id="action-box">
+            <span id="purgeCont" class="button btn-red" role="button">Delete and opt-out</span>
+            <span id="purgeAbort" class="button btn-gray" role="button">Opt-out later</span>
```

Comment:
Might sound a bit defensive? "Don't opt-out (now)" might do?

Reply:
Fine by me. I changed it.